src/js/stores/chat.js

```diff
@@ -11,12 +10,20 @@ export default Reflux.createStore({
   listenables: ChatActions,
 
   onCreate(initiator, ...participants) {
-    let profileUrl = (username) => `${settings.endpoint}/${username}/profile/card#me`
-    chatAgent.createConversation(profileUrl(initiator), participants.map(profileUrl))
+    console.log('onCreate initiator',initiator,'participants',participants)
+    chatAgent.createConversation(initiator, participants)
       .then(create.completed)
   },
 
   onCreateCompleted(conversation) {
+    console.log('create completed with conversation', conversation)
     this.trigger(conversation)
+    
+    // @TODO @FIXME
+    window.location.href='#/conversations/' + conversation.id
```

Comment:
@Pagebakers Couldn't find a way to redirect to URL/route "properly". Could you fix this?